platforms/esp32_sfoc/Cargo.toml

```diff
@@ -19,6 +19,10 @@ esp-backtrace = { version = "0.14.2", features = [
     # "println",
     # "uart"
 ] }
+
+critical-section = "1.2.0"
+
+
```

Comment:
This isn't needed for now, I don't think, but will become relevant soon enough, so let's leave it in.